configs/default/GEPR-GEPRCF722BT.config

```diff
@@ -58,8 +58,8 @@ timer B03 AF1
 # pin B03: TIM2 CH2 (AF1)
 
 # dma
-dma ADC 3 1
-# ADC 3: DMA2 Stream 1 Channel 2
+dma ADC 3 0
```

Comment:
@SteveCEvans: Another target that was changed back to a brolen SPI DMA configiuration - do you have time to figure out how this target can be fixed?

@haslinghuis: Please do not merge problematic changes like this.

Reply:
I ckeched with them, they said that change the ADC dma work fine on 4.2. But with default define it will broken on 4.2, still doesn't know why.